TestModule/TestFunctions.cpp

```diff
@@ -1,5 +1,4 @@
 #include "TestFunctions.h"
-#include "CountFilesByTypeInDirectory.h"
```

Comment:
This function has not been used within the code base for a while now. Do we want to keep it around / move it for future use or just toss it? I'm fine with either choice...

Reply:
I'm fine with just tossing it. It would still be a part of history if we ever wanted to resurrect it.

Actually, the same thing may have happened to the `ParseCsv` and associated `Split` and `Trim` code. The INI file updates that reads the keys from a section replaces their only use.

Reply:
The trim functions are fairly robust and we'll written. Tough to just toss them.

Reply:
I know, especially with all the time and thought I put into them. But they are still part of history, so easy to bring back if we ever need to.

Might be good to get rid of them so new users aren't confused by all that code that's never used.

Reply:
OK, deleted the function.